tools/complex_permissions_test.py

```diff
@@ -25,9 +25,7 @@ def _run_deno(self, flags, args):
         return tty_capture(cmd, b'')
 
 
-class TestReadPermissions(BaseComplexPermissionTest):
-    test_type = "read"
-
+class BaseReadWritePermissionsTest(BaseComplexPermissionTest):
```

Comment:
How does this work without test_type?
Should be called ReadWritePermissionsTestMixin and subclass object?

Reply:
You're right, fixing